exercises/grains/grains_test.rb

```diff
@@ -1,45 +1,81 @@
-#!/usr/bin/env ruby
-gem 'minitest', '>= 5.0.0'
 require 'minitest/autorun'
 require_relative 'grains'
 
+# Test data version: aa12f2e
 class GrainsTest < Minitest::Test
-  def test_square_1
+  def test_1
+    # skip
     assert_equal 1, Grains.square(1)
   end
 
-  def test_square_2
+  def test_2
     skip
     assert_equal 2, Grains.square(2)
   end
 
-  def test_square_3
+  def test_3
     skip
     assert_equal 4, Grains.square(3)
   end
 
-  def test_square_4
+  def test_4
     skip
     assert_equal 8, Grains.square(4)
   end
 
-  def test_square_16
+  def test_16
     skip
-    assert_equal 32_768, Grains.square(16)
+    assert_equal 32768, Grains.square(16)
```

Comment:
I elected to use the`32769` notation rather than `32_768` for simplicities sake. Is this okay, or should I update it to use the other notation?